.buildkite/pipelines/pull_request/response_ops.yml

```diff
@@ -7,7 +7,7 @@ steps:
       - build
       - quick_checks
     timeout_in_minutes: 120
-    parallelism: 4
+    parallelism: 7
```

Comment:
due to the typo in the config tests were not running, so that's why I had to increase it